src/Server/WPHelper.php

```diff
@@ -46,22 +50,79 @@ public function parseRequestParams( $method, array $bodyParams, array $queryPara
 		 * persisted queries (and ends up being a bit more flexible than
 		 * graphql-php's built-in persistentQueryLoader).
 		 *
-		 * @param array $data An array containing the pieces of the data of the GraphQL request
-		 * @param array $request_context An array containing the both body and query params
+		 * @param array  $parsed_body_params An array containing the pieces of the data of the GraphQL request.
+		 * @param array  $request_context    An array containing the both body and query params.
+		 * @param string $method             The method of the request (GET, POST, etc).
 		 */
+		$parsed_body_params = apply_filters( 'graphql_request_data', $parsed_body_params, $request_context, $method );
```

Comment:
Not sure if this is a flaw with the _old_ code (using the filter to modify `$parsed_body_params` and `$parsed_query_params`), but this change means `$parsed_query_params` doesnt get a filter at all.

Reply:
The `$request_context` holds the data from both body and query parameters. I remember not adding a different hook only because this data is already available here.

Reply:
Is it still worth adding it in a separate hook? Reading the code, I didn't see how useful this would be introducing a new hook before this action is run.